platooning_strategic_IHP/src/platoon_manager_ihp.cpp

```diff
@@ -270,12 +272,18 @@ namespace platoon_strategic_ihp
     int PlatoonManager::allPredecessorFollowing(){
 
         ///***** Case Zero *****///
-        // If the host vehicle is the second vehicle in this platoon,we will always follow the platoon leader in front of host vehicle
-        if(hostPosInPlatoon_ == 1) {
-            ROS_DEBUG_STREAM("As the second vehicle in the platoon, it will always follow the leader. Case Zero");
-            return 0;
+        // If the host vehicle is the second follower of a platoon, it will always follow the platoon leader in the front 
+        if(getNumberOfVehicleInFront() == 1) {
+            if (isFollower){
```

Comment:
I think this case 0 is just a quick bypass for the test when there is only 2 vehicles.  Since the current  test involves more then two vehicles, I feel like the entire case 0 could be removed, as the rest of the APF works just fine without it.

Reply:
removed the isFollower condition.